docs/resource-policy/policy/balloons.md

```diff
@@ -99,8 +101,15 @@ Balloons policy parameters:
   value set here is the default for all balloon types, but it can be
   overridden with the balloon type specific setting with the same
   name.
-- `balloonTypes` is a list of balloon type definitions. Each type can
-  be configured with the following parameters:
+- `balloonTypes` is a list of balloon type definitions. The order of
+  the types is significant in balloon CPU allocation and in choosing a
+  balloon type for a new container. The first type with the highest
+  `allocatorPriority` will get CPUs before other balloons when their
+  `minBalloons` > 0. A new container without balloon annotations will
```

Comment:
Just a note that should we come up with a term for these "minBalloons > 0" balloons. "pre-allocated" or "pre-inflated" or "persistent" or "non-volatile" or "sustained" or smth. So we could discuss the division between the two types of balloons. WDYT?

Reply:
Good point. Perhaps in a follow-up PR.

Reply:
@marquiz, I agree.